vms/platformvm/txs/mempool/mempool.go

```diff
@@ -224,26 +233,33 @@ func (m *mempool) HasTxs() bool {
 }
 
 func (m *mempool) PeekTxs(maxTxsBytes int) []*txs.Tx {
```

Comment:
Ideally this function should be removed entirely (Not really an issue for this PR - just making sure we know there is more work needed here going forward)